explain-indexes.md

```diff
@@ -0,0 +1,343 @@
+
+## Point_Get and Batch_Point_Get
+
+The `Point_Get` and `Batch_Point_Get` operator is used when retrieving data directly from a Primary Key or Unique Key. This operator is more efficient than `IndexLookup`. For example:
+
+{{< copyable "sql" >}}
+
+```sql
+EXPLAIN SELECT * FROM t1 WHERE id = 1234;
+EXPLAIN SELECT * FROM t1 WHERE id IN (1234,123);
+
+ALTER TABLE t1 ADD unique_key INT;
+UPDATE t1 SET unique_key = id;
+ALTER TABLE t1 ADD UNIQUE KEY (unique_key);
+
+EXPLAIN SELECT * FROM t1 WHERE unique_key = 1234;
+EXPLAIN SELECT * FROM t1 WHERE unique_key IN (1234, 123);
+```
+
+```sql
++-------------+---------+------+---------------+---------------+
+| id          | estRows | task | access object | operator info |
++-------------+---------+------+---------------+---------------+
+| Point_Get_1 | 1.00    | root | table:t1      | handle:1234   |
++-------------+---------+------+---------------+---------------+
+1 row in set (0.00 sec)
+
++-------------------+---------+------+---------------+-------------------------------------------------+
+| id                | estRows | task | access object | operator info                                   |
++-------------------+---------+------+---------------+-------------------------------------------------+
+| Batch_Point_Get_1 | 2.00    | root | table:t1      | handle:[1234 123], keep order:false, desc:false |
++-------------------+---------+------+---------------+-------------------------------------------------+
+1 row in set (0.00 sec)
+
+Query OK, 0 rows affected (0.27 sec)
+
+Query OK, 1010 rows affected (0.06 sec)
+Rows matched: 1010  Changed: 1010  Warnings: 0
+
+Query OK, 0 rows affected (0.37 sec)
+
++-------------+---------+------+----------------------------------------+---------------+
+| id          | estRows | task | access object                          | operator info |
++-------------+---------+------+----------------------------------------+---------------+
+| Point_Get_1 | 1.00    | root | table:t1, index:unique_key(unique_key) |               |
++-------------+---------+------+----------------------------------------+---------------+
+1 row in set (0.00 sec)
+
++-------------------+---------+------+----------------------------------------+------------------------------+
+| id                | estRows | task | access object                          | operator info                |
++-------------------+---------+------+----------------------------------------+------------------------------+
+| Batch_Point_Get_1 | 2.00    | root | table:t1, index:unique_key(unique_key) | keep order:false, desc:false |
++-------------------+---------+------+----------------------------------------+------------------------------+
+1 row in set (0.00 sec)
+```
+
+## IndexFullScan
+
+Because indexes are ordered, the `IndexFullScan` can be used to optimize common queries such as the `MIN` or `MAX` values for an indexed value:
```

Comment:
```suggestion
Because indexes are ordered, the `IndexFullScan` operator can be used to optimize common queries such as the `MIN` or `MAX` values for an indexed value:
```